src/components/tiles/table/use-title-size.ts

```diff
@@ -29,13 +29,15 @@ export const useTitleSize = ({ readOnly, columns, dataSet, measureColumnWidth, r
                       1 - (readOnly ? 0 : kControlsColumnWidth));
     };
     rowChanges; // eslint-disable-line no-unused-expressions
-    return getTitleCellWidthFromColumns();
+    const widthFromColumns = getTitleCellWidthFromColumns();
+    return widthFromColumns < kDefaultColumnWidth ? kDefaultColumnWidth : widthFromColumns;
   }, [readOnly, columns, dataSet, measureColumnWidth, rowChanges]);
 
   const getTitleHeight = useCallback(() => {
     const font = `700 ${defaultFont}`;
     const lines = measureTextLines(dataSet.name || 'Table 8', titleCellWidth - 2 * kTitlePadding, font);
-    return lines * kCellLineHeight + 2 * kTitlePadding;
```

Comment:
It looks like the value of `lines` here is always `1`, because as long as the source data defaults to the table, the `dataSet.name` is going to be something short (while the table title could be long).  This is just to point out that this PR does not introduce that behavior, but it doesn't fix it either.